### DISCUSSION TITLE
Problems with Wordpress after migrate to EasyEngine

### DISCUSSION THREAD

**Discussion by schrutcl** [EasyEngine/easyengine#1773] [Support Requests]
Greetings,

I did the migration from ServerPilot to EasyEngine and now I have problems in Wordpress, more precisely the Health Screen where it shows me various errors and I think the configuration is not completely correct.

1) First I got a new server
2) I transferred my website to a new server in a .zip and .sql file, ie Wordpress and Database.
3) Then installed EasyEngine on it with the command "wget -qO ee rt.cx/ee4 && sudo bash ee"
4) Then I created the website with the command "sudo ee site create mywebsite.com --type=wp --cache --ssl=le"
5) I deleted the Wordpress installation that EasyEngine installed and uploaded my previous Wordpres site from the .zip file.
6) I deleted the database that was there before and inserted my own database.
7) I also edited all the details in the wp-config and moved the wp-config from the folder where the wordpress installation is located to a folder outside of htdocs.
8) I also changed the folder and file permissions so that the site is visible and everything works.
9) I noticed that when EasyEngine installs wordpress in wp/content there is object-cache.php but it is not there in my wordpress, I decided to put that file in wp-content but then I have a problem that wordpress in the administrative panel does not allow me to change anything, even the permalinks are changed to something else and it is also impossible to update any changes.

Can someone help me to solve this problem because I think that it is currently affecting the performance and the overall business because I also felt a drop in traffic.

Thanks in advance!

![Wordpress - Site Health](https://user-images.githubusercontent.com/124939000/217955756-a7b44271-d414-465a-b223-f581d5ffb8c1.jpg)

**Comment by JustDr00py** [comment]
try installing WP Redis Plugin |    Version 1.3.2 | By Pantheon, Josh Koenig, Matthew Boynes, Daniel Bachhuber, Alley Interactive

and nginx-helper Plugin | Version 2.2.2 | By rtCamp
and set to redis cache.

**Reply by JustDr00py** [reply]
you probably also have to set the hostname to the redis docker container.

**Reply by schrutcl** [reply]
Hello JustDr00py, thank you for your reply!

I installed WP Redis and also nginx-helper, but the problem is still not solved.

I don't know why this happens, after I set up object-cache.php or convert it to a symlink that points to where wp-redis is installed, then wordpress finds itself strange and it's absolutely impossible to update anything type of settings in wordpress.

**Reply by schrutcl** [reply]
I don't know what exactly happened, but I solved one problem. I added object-cache.php using WP-CLI. And the problem "You should use a persistent object cache" has disappeared, but these others are still there, is there any recommendation on what I should do?

![Screenshot_1](https://user-images.githubusercontent.com/124939000/218253665-fb4dd43e-f8b7-4217-ba26-d24dbf24a927.jpg)
